### ISSUE TITLE
[react-router example] Upgrade to react-router 1.x

### ISSUE THREAD

**Issue by mridgway** [yahoo/fluxible#342]

**Comment by hfter789** [comment]
hi @mridgway 
I was trying to upgrade the react-router example to use react-router 1.x
But I keep getting a warning: 

> Warning: Failed propType: Required prop `context` was not specified in `ApplicationContextProvider`. Check the render method of `RoutingContext`.

which becomes an error later:

>TypeError: Cannot read property 'executeAction' of undefined
    at ContextProvider.Object.assign.getChildContext (/Users/hfter789/mycode/fluxible/examples/node_modules/fluxible-addons-react/provideContext.js:34:50)

The server I have currently looks like this:
````
server.use(function (req, res, next) {
    debug('Executing navigate action');
    match({
        routes: app.getComponent(),
        location: req.url
    }, function (error, redirectLocation, renderProps) {
        if (error) {
            res.status(500).send(error.message);
        } else if (redirectLocation) {
            res.redirect(302, redirectLocation.pathname + redirectLocation.search);
        } else if (renderProps) {
            var context = app.createContext();
            context.executeAction(navigateAction, {path: req.url}, function () {
                debug('Exposing context state');
                var exposed = 'window.App=' + serialize(app.dehydrate(context)) + ';';
                var markupElement = React.createElement(
                        FluxibleComponent,
                        { context: context.getComponentContext() },
                        React.createElement(RoutingContext, renderProps)
                    );
                var html = renderToStaticMarkup(HtmlComponent({
                    context: context.getComponentContext(),
                    state: exposed,
                    markup: renderToString(markupElement)
                }));

                debug('Sending markup');
                res.status(200).send(html);
            });
        } else {
            next();
        }
    })
});
```

Can you give me some insights on what might be causing this?

**Comment by mridgway** [comment]
Sure. Can I reproduce this just using the PR branch you have?

**Comment by hfter789** [comment]
@mridgway I just updated my PR. You should be able to reproduce the issue with it. Thanks!

**Comment by mridgway** [comment]
Yup, still looking into this.

**Comment by mridgway** [comment]
I created a PR to your branch with some fixes. I *think* it's working correctly now.

**Comment by mridgway** [comment]
Resolved in #364
